### DISCUSSION TITLE
Need cognito access and refresh token after connecting to web3 auth

### DISCUSSION THREAD

**Discussion by irfanonk** [Web3Auth/Web3Auth#1008] [Help]
Hi,

I am connecting to web3 auth using cognito idToken and able to access user details web3auth.getUserInfo(). But I also need access and refresh tokens from cognito using web3auth.getUserInfo(). Is it possible to include these attributes into user info?

Thanks

**Comment by shahbaz17** [comment]
Hey @irfanonk 

`web3auth.getUserInfo()` returns an `oAuthAccessToken` which is the same returned from Cognito in your case here. But it doesn't have refresh tokens.

May I know your use case here?
